profile.d/00-pengwin.sh

```diff
@@ -1,5 +1,14 @@
-# enable external x display
-export DISPLAY=:0
+# check whether it is WSL1 for WSL2 by using gcc version kernel built with
+if[ $(grep -oE 'gcc version ([0-9]+)' /proc/version | awk '{print $3}') -gt 5 ]; then
+  # enable external x display for WSL 2
+  ipconfig_exec="C:\\Windows\\System32\\ipconfig.exe"
```

Comment:
Is it now better to use which ipconfig.exe ? Windows may have been installed in D:

Reply:
The issue here is that `which ipconfig.exe` will be useless if the user chooses not to include the Windows environment path. Should we use something like `find {the drive mounted} -name ipconfig.exe`?